aws/autoscaling.go

```diff
@@ -18,10 +20,51 @@ import (
 	"github.com/mozilla-services/reaper/state"
 )
 
+type AutoScalingGroupScalingSchedule struct {
+	enabled           bool
+	scaleDownString   string
+	scaleUpString     string
+	previousScaleSize int64
+	previousMinSize   int64
+}
+
+func (s *AutoScalingGroupScalingSchedule) setSchedule(tag string) {
+	// scalerTag format: cron format schedule (scale down),cron format schedule (scale up),previous scale time,previous desired size,previous min size
+	splitTag := strings.Split(tag, ",")
+	if len(splitTag) != 4 {
+		log.Error("Invalid Autoscaler Tag format %s", tag)
+	} else {
+		prev, err := strconv.ParseInt(splitTag[2], 0, 64)
+		if err != nil {
+			log.Error(err.Error())
+		}
+		min, err := strconv.ParseInt(splitTag[3], 0, 64)
+		if err != nil {
+			log.Error(err.Error())
+		}
+		s.scaleDownString = splitTag[0]
+		s.scaleUpString = splitTag[1]
+		s.previousScaleSize = prev
+		s.previousMinSize = min
+		s.enabled = true
```

Comment:
You should probably not fall through to here in the case of prev/min conversion errors.